### ISSUE TITLE
undefined method `new_session_path'

### ISSUE THREAD

**Issue by jonhattan** [doorkeeper-gem/doorkeeper-devise-client#8]
I'm gettting this error after signin in with oauth2 provider, in `/users/auth/doorkeeper/callback`:

```
undefined method `new_session_path' for #<Users::OmniauthCallbacksController:0x000000042d2978>
```

I've followed some tricks in https://github.com/plataformatec/devise/issues/1390 with no success.

Any idea?

**Comment by qnm** [comment]
@jonhattan I'm seeing the same issue - did you get to the bottom of it?

**Comment by romanvbabenko** [comment]
+1

**Comment by doersf** [comment]
+1

**Comment by matthewsinclair** [comment]
I notice this issue has been around for a while. Has anyone had any luck with it? I'm (still) getting the same thing.

**Comment by dredupuika** [comment]
all explained in the [https://github.com/plataformatec/devise/wiki/OmniAuth:-Overview](devise wiki)

**Comment by felipeelias** [comment]
closed by 5afaf47
